packages/arb-token-bridge-ui/src/components/TransferPanel/TokenRow.tsx

```diff
@@ -136,18 +161,24 @@ function TokenListInfo({ token }: { token: ERC20BridgeToken | null }) {
 
 interface TokenRowProps {
   style?: React.CSSProperties
-  onTokenSelected: (token: ERC20BridgeToken | null) => void
-  token: ERC20BridgeToken | null
+  onTokenSelected: (parentErc20Address: string | null) => void
```

Comment:
we only use token address from `ERC20BridgeToken` so we only pass that now, making it easier to work with